yw_website/yw_website/apps/website/templates/pages/my_workflows.html

```diff
@@ -1,5 +1,6 @@
-{% extends 'extendable/_base_workflow_listview.html' %}
+{% extends 'extendable/_base_my_workflow_listview.html' %}
```

Comment:
Could probably just add an if statement so we can reuse the list view with an added edit button